### DISCUSSION TITLE
Does flow launcher have ability to open a program in a new window/monitor

### DISCUSSION THREAD

**Discussion by mx3214** [Flow-Launcher/Flow.Launcher#2336] [Q&A] [answered]
Hi,

I work on a dual monitor display.

Often I reading a website, and would like to search something on google. 

If I open Flow Launcher and type in my query, it loads the result as a new tab on the same display monitor. This is problematic as it distracts me from the place I was reading on the original website.

Is it possible to get flow launcher to open my browser in a new window, on my 2nd display with the search query? I know I could just open a new browser window, move it to the 2nd display and then open flow launcher on the second display and type in my search query. It's not a huge issue to do this, but it is a bit long winded.

Thanks in advance

**Comment by jjw24** [comment]
Hi there, check flow's settings, there is an option to open url as a new tab/window. I don't remember what it's called right now, should be in the 'General' section.

**Reply by jjw24** [reply, marked as answer]
I just reread your question actually, you are after the ability to open the url as a window on a second display monitor, we actually don't have that functionality yet

**Reply by mx3214** [reply]
Opening a url in a in a new window is still quite useful. 

I see how you can set it permanently as a new tab/window. 

However is it possible to have it non permanent & change on each search depending on what I want? Eg some hotkey to have it open as a new tab on one occasion, but on another occasion have it open in a new window with some different hotkey.

**Reply by jjw24** [reply]
Unfortunately we don't have that feature, best to open an issue for it.

**Reply by mx3214** [reply]
Okay thanks, one last related question. Is it possible to open up a program in the background? As opposed to the foreground.

**Reply by jjw24** [reply]
That's also something we haven't implemented.
